iree/compiler/Dialect/VMLA/Transforms/test/pre_conversion_lowering.mlir

```diff
@@ -31,3 +31,15 @@ func @f(%arg0: tensor<3xf32>) -> tensor<5x6x3xf32> {
   %0 = "mhlo.broadcast"(%arg0) {broadcast_sizes = dense<[5, 6]> : tensor<2xi64>} : (tensor<3xf32>) -> tensor<5x6x3xf32>
   return %0 : tensor<5x6x3xf32>
 }
+
+// -----
+
+// CHECK-LABEL: func @f
+func @f(%arg0: tensor<3xf32>, %arg1: tensor<3xf32>) -> tensor<3xf32> {
+  // CHECK-NOT: "mhlo.complex"
```

Comment:
Worth checking anything else here? Or is "doesn't contain `mhlo.complex`" enough? Should the `3xcomplex<f32>` types go away too?

Reply:
I added more checks for the specific structure. The primary goal is to optimize away the complex specific behavior by decomposing into real valued operations.